### PR TITLE
TC_00.001.04 | Creating new item > Creating Freestyle project

### PR DESCRIPTION
**Pull request by sergeykazak**
https://trello.com/c/3Kp9gtFQ/153-tc0000104-creating-new-item-creating-freestyle-project

### CODE REVIEW

File: src/test/java/school/redrover/FreestyleProject4Test.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+@Test
+public class FreestyleProject4Test extends BaseTest {
+
+
+    @Test
+    public void testFreestyleProject() {
```
Comment (bzhumanova) [inline src/test/java/school/redrover/FreestyleProject4Test.java:14]: I would rename the test name to create()

**Review by bzhumanova — APPROVED** [review]

File: src/test/java/school/redrover/FreestyleProject4Test.java (hunk 1 above)
Comment (sergeykazak) [reply src/test/java/school/redrover/FreestyleProject4Test.java]: Thanks for the comment @bzhumanova ! fixed in the new commit: https://github.com/RedRoverSchool/JenkinsQA_07/pull/1067/commits/a05a6e627cbbb4492b1c4ff97deaebb13d380dc2 please review.

File: src/test/java/school/redrover/FreestyleProject4Test.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package school.redrover;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+@Test
```
Comment (VictoriaLema) [inline src/test/java/school/redrover/FreestyleProject4Test.java:9]: Hi! I don't think @Test should be written before class

Comment (sergeykazak) [reply src/test/java/school/redrover/FreestyleProject4Test.java]: Thanks a lot for this finding! fixed in https://github.com/RedRoverSchool/JenkinsQA_07/pull/1067/commits/05c7cec8e28c94ff2872fa732b7333239f7e01f8

**Review by SafronovaOlga — APPROVED** [review]

**Pull request merged**